problem2.py

```diff
@@ -1 +1,15 @@
+"""
+Name:   problem2.py
 
+Purpose: This problem is meant for determining how to fairly split the chicken that was won amongst the class and Mr. Fabroa.
+
+Author: Au.E
+
+Date: 07/12/2020
+"""
+#input data for amount of chicken
+chicken = int(input("how many pieces of chicken are there?"))
```

Comment:
put a space after your input prompt for better user interface `"how many pieces of chicken are there? "`
also start your prompt with a capital letter